packages/types/src/lib/merch.ts

```diff
@@ -80,6 +80,14 @@ export interface Promotion {
   }>;
 }
 
+export interface PromoInfo {
+  
+  promotion_id: string;
+  name: string;
+  discount_percentage: string;
+  category: string;
+}
+
```

Comment:
https://github.com/ntuscse/website/blob/main/packages/types/src/lib/merch.ts

great job but could you refer to the exisitng Promo type in merch.ts types